.github/workflows/codeql.yml

```
@@ -84,7 +82,4 @@ jobs:
         CI: "true"
 
     - name: Perform CodeQL Analysis
-      uses: github/codeql-action/analyze@v3
-      with:
-        category: "/language:${{ matrix.language }}/advanced"
-        upload: true
\ No newline at end of file
+      uses: github/codeql-action/analyze@v3
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)